usagov/vote-gov-drupal
# VOTE-1463 Remove accordion group block permissions from content manage…

…r role

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-1463

## Description (optional)
Remove deprecated accordion group permission from Content Manager role.
Also fix 404 image in console logo_square_VoteGov_Logo2.png 

## Deployment and testing (required, if applicable)
### Post-deploy steps
1. run `lando retune`

### Testing steps
1. No test as block type: accordion group has already been removed from the system.
2. Visit http://vote-gov.lndo.site/ and make sure the console log is clear
